packages/odyssey-design-tokens/src/space.json

```diff
@@ -1,13 +1,13 @@
 {
   "space": {
-    "fixed": {
```

Comment:
rem units are never fixed, they are root relative. I'm changing the naming structure here to avoid confusion and offer more clarity when referencing these variables in our styles

Reply:
We've gotten requests in the past not to refer to things by unit, but this approach is clearer.

IMO ff someone is going to be designing or implementing with these values, they should understand which units are in play and how they work.